Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2735,12 +2735,13 @@ def gen2018HiMix(fragment,howMuch):
                           '--era' : 'Run3'
                           }
 
-steps['HARVESTFS']={'-s':'HARVESTING:validationHarvesting',
+steps['HARVESTFS']={'-s':'HARVESTING:validationHarvestingFS',
```

Comment:
@jfernan2 , just wondering, what about these two lines?
https://github.com/cms-sw/cmssw/blob/1d6c40888e62e3413754891fad82a068df5cd1be/Configuration/PyReleaseValidation/python/relval_steps.py#L2819
https://github.com/cms-sw/cmssw/blob/1d6c40888e62e3413754891fad82a068df5cd1be/Configuration/PyReleaseValidation/python/relval_steps.py#L3230

Reply:
@chayanit good point! As you can see from the commits validationHarvestingFS is removing HLT and Tau modules. So I only changed FastSim Run1 workflows since I found there HLT and Tau empty histograms (wf 5.1), in HARVESTUP15FS (wf 135.4) HLT and Tau are not present hence I did not dare to do the same.
If you think this would be good for coherence I am willing to change them too. 
In any case, it would be good to seat PdmV and DQM around one table to start an uniformity of the Validation packages and sequences, it seems to me Validation is living between two worlds. Thanks

Reply:
@jfernan2 hmm ok, then I think we can leave them as they are for now. I'm also not sure if they will be affected. And yes, we can try to organise a round table discussion between DQM and PdmV.